### ISSUE TITLE
Questions about the Competition

### ISSUE THREAD

**Issue by jiafongyeh** [IBM/cdfsl-benchmark#4]
Hi @yunhuiguo .

Thank you for hosting this competition, it's really interesting and challenging!
After carefully reading the rules on your website and in your paper, I still have some questions, please correct me if I misunderstood.

1.  Is it allowed to use the paper submitted to your workshop as the paper required in this competition? 
2.  Is it allowed to modify those hyper-parameters for training? (e.g.  epoch, n_query)
Intuitively, training a module by the normal way will have a better performance than by the meta-train way in current setting. (baseline will see more images)

In one epoch:
baseline will see: 16 (batch size) * 2400 (dataloader length) = 38400 images
protonet will see: (5+16)(n_support + n_query) * 5 (n_way) * 100 (dataloader length) = 10500 images

3. During testing (in finetune.py), it seems only concat the trained backbone with a learnable classifier.
Other mechanisms in the module will not affect, right? (e.g. calculate the prototype in PN, relation module in RN).
Or we can modify the code, let other mechanisms also work during testing?

Looking forward to your reply, thank you so much.

**Comment by yunhuiguo** [comment]
Hi @JiaFong 
1. Yes, it is fine.  But you need to submit your paper both to the challenge track and the workshop track.
2. It is fine to modify the hyper-parameters such as epoch, learning rate related to the training process but not n_query. And the detailed hyper-parameters need to be specified in the paper.
3. Not very sure about your question. You are free to modify the classification algorithms.

**Comment by jiafongyeh** [comment]
Hi, thank you for your reply.

Sorry about the question 3, I didn't noticed that we can modify the classification algorithm.
Is there any other restriction we need to followed during fine-tune stage? 

(e.g. in the current codes, the linear classifier will be re-created when a new task coming. is it allowed to use the same classifier during whole testing? )


Thank you.

**Comment by yunhuiguo** [comment]
Hi @JiaFong , it is not allowed to use the same classifier during whole testing. Each episode during testing should be viewed as a different task. Generally, you are free to modify the classification algorithm, but the evaluation settings (number of query examples, number of support examples, etc.) should stay the same.

**Comment by jiafongyeh** [comment]
Hi @yunhuiguo .

So the backbone、classifier algorithm need to be reset at the beginning of each test episode.
No information can be recorded from the previous task to the new one, right?

**Comment by yunhuiguo** [comment]
@JiaFong That's right. Each trial (or episode) should be an independent task.

**Comment by jiafongyeh** [comment]
Ok, thank you for your explanation.
